source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -182,7 +182,7 @@ namespace PlayFab
             eventContents.Payload = value;
             request.Events.push_back(eventContents);
 
-            eventsApi->WriteTelemetryEvents(request, WriteEventsSuccessCallBack, WriteEventsFailureCallBack);
+            eventsApi->WriteEvents(request, WriteEventsSuccessCallBack, WriteEventsFailureCallBack);
```

Comment:
Why this change? Is the WriteTelemetryEventsAPI missing from Azure? We should discuss more, if so, as that may be in error.